src/components/WelcomePage.tsx

```diff
@@ -5,16 +5,16 @@ import Navbar from "./Navbar";
 import Button from "./Button";
 import HeroSection from "./HeroSection";
 import "../stylesheets/HomepageRow.css";
-import React, { MouseEventHandler } from "react";
+import React from "react";
 import { ROUNDED_QUESTION_COUNT } from "../constants";
+import { NavLink } from "react-router-dom";
```

Comment:
same issue here.
If NavLink is not being used here, then this import is not necessary